angr/analyses/cfg_accurate.py

```diff
@@ -990,7 +990,8 @@ def _process_hints(self, analyzed_addrs, remaining_entries):
                                           new_state,
                                           self._context_sensitivity_level
                                           )
-                remaining_entries.append(new_path_wrapper)
+                self._insert_entry(new_path_wrapper)
```

Comment:
This change is not what I was thinking when writing that piece of code. Addresses from function hints should have a lower priority than any existing jobs (or `entries` as in the current code base), since those hints are not always reliably pointing to functions. They may very well be random addresses that point to the middle of instructions.